index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Tranquil Essence Spa</title>
+    <!-- Favicon-->
+    <link rel="icon" href="images/logo.png" sizes="16x16" type="image/png" />
+    <!-- Link to CSS -->
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <!-- Hero Section -->
+    <header>
+      <div class="hero-section">
+        <!-- Video background with fallback image for unsupported browsers -->
+        <video autoplay muted loop class="hero-video">
+          <source src="video/spa.mp4" type="video/mp4" />
+        </video>
+        <!-- Fallback image for browsers that don't support the video tag -->
+        <img
+          src="images/spa.png"
+          alt="Tranquil Essence Spa"
+          class="fallback-image"
+        />
+      </div>
+      <!-- Logo of the Spa -->
+      <div class="logo">
+        <img src="images/logo.png" alt="Tranquil Essence Spa Logo" />
+      </div>
+
+      <!-- Navigation Section -->
+      <div class="navigation">
+        <!-- Button to toggle navigation menu -->
+        <button class="menu-toggle">Menu</button>
+      </div>
+
+      <!-- Hero Content (Title and Tagline)-->
+      <div class="hero-content">
+        <h1>Tranquil Essence Spa</h1>
+        <p><em>Relax, Rejuvenate, Renew</em></p>
+      </div>
+    </header>
+
+    <!-- Signup Section -->
+    <section class="signup">
+      <form action="https://httpbin.org/anything" method="POST">
+        <h2>Sign up</h2>
+        <fieldset>
+          <!-- Name Input Field -->
+          <label for="name"></label>
+          <input
+            type="text"
+            id="name"
+            name="name"
+            placeholder="Name"
+            required
+          />
+          <!-- Email Input Field -->
+          <label for="email"></label>
+          <input
+            type="email"
+            id="email"
+            name="email"
+            placeholder="E-mail"
+            required
+          />
+          <!-- Password Input Field -->
+          <label for="password"></label>
+          <input
+            type="password"
+            id="password"
+            name="password"
+            placeholder="Password"
+            required
+          />
+          <!-- Radio Buttons for Time Selection -->
```

Comment:
I think it might be common practice to include a label before radio buttons on an appointment request form to improve clarity, something like "Preferred appointment time" or "Select your preferred time slot"